Intersect.Editor/Forms/Editors/Events/CommandPrinter.cs

```diff
@@ -1076,7 +1076,7 @@ private static string GetCommandText(PlayAnimationCommand command, MapInstance m
                             AnimationBase.GetName(command.AnimationId),
                             Strings.EventCommandList.animationonmap.ToString(
                                 MapList.OrderedMaps[i].Name, command.X, command.Y,
-                                Strings.Directions.dir[(sbyte) command.Dir]
+                                Strings.Direction.dir[(Direction) command.Dir]
```

Comment:
On the other lines you consume command.Dir directly, why is this cast needed?

Reply:
the "Dir" field isn't really used as `Direction` by the PlayAnimationCommand EventCommand

![imagen](https://user-images.githubusercontent.com/17498701/223057905-461eb39b-0b0a-40cf-992b-3eee06dc884e.png)


![imagen](https://user-images.githubusercontent.com/17498701/223057114-7b54ff14-7219-4197-95ef-9f27420456f3.png)

Reply:
Oh, so command.Dir is direction _sometimes_ but not _all the time_?

Reply:
yeah, apparently is used differently in this one case ( PlayAnimationCommand )
this should be an enum instead of "Dir" to be honest, but i don't think that we should go into that now o.O